autoload/lsp/completion.vim

```diff
@@ -137,6 +137,57 @@ def CompletionUltiSnips(prefix: string, items: list<dict<any>>)
   endfor
 enddef
 
+# Integration with the vim-vsnip plugin
+def CompletionVsnip(items: list<dict<any>>)
+  def Pattern(abbr: string): string
+    var chars = escape(abbr, '\/?')->split('\zs')
+    var chars_pattern = '\%(\V' .. chars->join('\m\|\V') .. '\m\)'
+    var separator = chars[0] =~ '\a' ? '\<' : ''
+    return separator .. '\V' .. chars[0] .. '\m' .. chars_pattern .. '*$'
```

Comment:
Can you change this to use string interpolation `$'{separator}\V{chars[0]}\m{chars_pattern}*$'`